extensions/commands/sbom/cmd_cyclonedx.py

```diff
@@ -102,7 +102,8 @@ def create_component(node) -> Component:
         component = Component(
             type=package_type_to_component_type(node.conanfile.package_type),
             name=node.name or f'UNKNOWN.{id(node)}',
-            author=node.conanfile.author if node.conanfile.author else "Conan",
```

Comment:
this was wrong in the first place: "Conan" is a tool, not a legal entity, and may not be a author nor supplier.